tests/backends/test_awkward.py

```diff
@@ -78,27 +78,25 @@ def test_projection():
 
     out = array.to_rhophietatau()
     assert isinstance(out, vector._backends.awkward_.VectorArray4D)
-    assert out.tolist() == [
-        [
-            {
-                "rho": 2.23606797749979,
-                "phi": 1.1071487177940904,
-                "eta": 1.1035868415601453,
-                "tau": 0,
-                "wow": 99,
-            }
-        ],
-        [],
-        [
-            {
-                "rho": 6.4031242374328485,
-                "phi": 0.8960553845713439,
-                "eta": 0.8361481196083128,
-                "tau": 0,
-                "wow": 123,
-            }
-        ],
-    ]
+    assert out.tolist()[0][0] == pytest.approx(
```

Comment:
Wondering if I knew `approx` worked on a dict. :) Anyway, this now doesn't test completeness, this would now pass if there was one more item. Maybe

`(a, b, c), = out.to_list()`, then check `a`, `b` and `c`?

Reply:
Sorry, I think I read that wrong, I think it's `(a,), b, (c,) =`.

Reply:
Oh yes, thanks!